src/Heading.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const Heading = () => {
+  return (
+    <div>
```

Comment:
This `<div>` wrapper isn't really necessary as everything is wrapped inside the `<header>` tags :)